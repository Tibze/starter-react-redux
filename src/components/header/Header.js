import React from 'react';
import {Link} from 'react-router';

import styles from './styles.scss';

import logo from '../../../static/assets/images/react-logo.png';
// import {connect} from 'react-redux';

export default class Header extends React.Component {
  render() {
    return (
      <div>
        <nav className="container nav">
          <div className="nav-left">
              <h1 className='title nav-title'>React / Redux</h1>
          </div>

          <span className="nav-toggle">
            <span></span>
            <span></span>
            <span></span>
          </span>

          <div className="nav-right nav-menu">
            <Link className='nav-item' to='/'>Home</Link>
            <Link className='nav-item' to='/about'>About</Link>
            <Link className='nav-item' to='/contact'>Contact</Link>
            <Link className='nav-item' to='/settings'>Settings</Link>
          </div>
        </nav>
    </div>
    );
  }
}
