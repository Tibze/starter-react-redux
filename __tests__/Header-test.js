
'use strict';

import React from 'react';
import Header from '../src/components/header/Header';
import renderer from 'react-test-renderer';

it('header renders correctly', () => {
  const header = renderer.create(
    <Header />
  ).toJSON();
  expect(header).toMatchSnapshot();
});
