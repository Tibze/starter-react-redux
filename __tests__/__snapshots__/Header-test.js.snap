// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header renders correctly 1`] = `
<div>
  <nav
    className="container nav"
  >
    <div
      className="nav-left"
    >
      <h1
        className="title nav-title"
      >
        React / Redux
      </h1>
    </div>
    <span
      className="nav-toggle"
    >
      <span />
      <span />
      <span />
    </span>
    <div
      className="nav-right nav-menu"
    >
      <a
        className="nav-item"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
      <a
        className="nav-item"
        onClick={[Function]}
        style={Object {}}
      >
        About
      </a>
      <a
        className="nav-item"
        onClick={[Function]}
        style={Object {}}
      >
        Contact
      </a>
      <a
        className="nav-item"
        onClick={[Function]}
        style={Object {}}
      >
        Settings
      </a>
    </div>
  </nav>
</div>
`;
